### PR TITLE
Contact Card, Contact Card Small Updates

### PR DESCRIPTION
**Pull request by Uri-E**
Updated Contact Card styles to match those of
standard faculty profile cards. Added link styles/icons, updated some text styles. 

If linked, name will have link icon and be orange. If not, won't have icon and be blue.

### CODE REVIEW

**Review by prd409-utsa — APPROVED** [review]

**Review by lalov1 — COMMENTED** [review]
Should we have a variant for the link? I'm not showing the name linked. Also, I always thought the `display: block` was a little weird on the email because the hover spans the entire width.

File: components/02-components/04-profile/02-contact-callout/contact-callout--small.hbs [hunk 1]
```diff
@@ -8,9 +8,16 @@
         <div class="col-12 col-sm-6 col-xl-8">
             <div class="staff-card-content">
                 <h4>Contact</h4>
```
[the remark this answers is not in the document]
Comment (Uri-E) [reply components/02-components/04-profile/02-contact-callout/contact-callout--small.hbs]: OK, I've added linked/unlinked variants and updated that code. Looks to be working out alright.

**Review by lalov1 — APPROVED** [review]
Looks good!

**Review by lalov1 — COMMENTED** [review]
👍

**Pull request merged**